tests/examples/test_cleanup_unwatched_media.py

```diff
@@ -0,0 +1,162 @@
+            return {
+                "statuses": {oid: {"status": 200} for oid in kwargs["data"]["oids"]}
+            }
+        elif url == "/stats/unwatched/":
+            return unwatched_tree
+        elif url == "/medias/resources-list/":
+            oid = kwargs["params"]["oid"]
+            return {"resources": all_resources.get(oid, {})}
+        elif url == "/medias/resources-delete/":
+            oid = kwargs["data"]["oid"]
+            if oid in all_resources.keys():
+                return {
+                    "success": True,
+                }
+            else:
+                return {"success": False}
+
+    from ms_client.client import MediaServerClient
+
+    client = MediaServerClient()
+    client.api = mock_api_call
+    return client
+
+
+@pytest.fixture()
+def all_resources():
+    return {
+        "v1234local": [
+            {
+                "manager": {"service": "local"},
+                "format": "m3u8",
+                "path": "media_1080_RB4YjJhKks.m3u8",
+                "file_size": 1000,
+            },
+            {
+                "manager": {"service": "local"},
+                "format": "m3u8",
+                "path": "media_720_yKViKrE37V.m3u8",
+                "file_size": 500,
+            },
+            {
+                "manager": {"service": "local"},
+                "format": "mp4",
+                "path": "media_1080_Gqi7RHDWdv.mp4",
+                "file_size": 1000,
+            },
+        ],
+        "v1234object": [
+            {
+                "manager": {"service": "object"},
+                "format": "m3u8",
+                "path": "media_1080_RB4YjJhKks.m3u8",
+                "file_size": 1000,
+            }
+        ],
+        "v1234youtube": [
+            {
+                "manager": {"service": "youtube"},
+            }
+        ],
+    }
+
+
+@pytest.fixture()
+def unwatched_tree():
+    return {
+        "success": True,
+        "start_date": "2023-09-10",
+        "end_date": "2023-09-11",
+        "unwatched": [
+            {
+                "object_id": "v1234local",
+                "storage_used": 2000000,
+            },
+            {
+                "object_id": "v1234object",
+                "storage_used": 1000000,
+            },
+            {
+                "object_id": "v1234youtube",
+                "storage_used": 1000000,
+            },
+            {
+                "object_id": "l12665a308e2b1mfwdke",
+                "storage_used": 9970,
+            },
+        ],
+    }
+
+
+def test_get_human_readable_size():
+    assert e.get_human_readable_size(1) == "1.0B"
+    assert e.get_human_readable_size(1000) == "1.0KB"
+    assert e.get_human_readable_size(1000000) == "1.0MB"
+    assert e.get_human_readable_size(1000000000) == "1.0GB"
+    assert e.get_human_readable_size(1000000000000) == "1.0TB"
+
+
+def test_filter_vod(unwatched_tree):
+    assert len(e.filter_vod(unwatched_tree["unwatched"])) == 3
```

Comment:
`assert set(vod['oid'] for vod in e.filter_vod(unwatched_tree["unwatched"]) == {...}`